Resources/Prototypes/Entities/Objects/Devices/Syndicate_Gadgets/reinforcement_teleporter.yml

```diff
@@ -25,7 +25,7 @@
   - type: GhostRole
     name: ghost-role-information-syndicate-reinforcement-spy-name
     description: ghost-role-information-syndicate-reinforcement-description
-    rules: ghost-role-information-rules-default-solo-antagonist
+    rules: ghost-role-information-syndicate-monkey-reinforcement-rules
```

Comment:
the normal reinforcment should not get the monkey rules

Reply:
With the new commit, all Syndicate reinforcements now get the same rule locale. The previous 3 all said the exact same thing